src/utils/components/SysprepModal/hooks/useConfigMaps.ts

```diff
@@ -0,0 +1,24 @@
+import { ConfigMapModel, modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1ConfigMap } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { useK8sWatchResource, WatchK8sResult } from '@openshift-console/dynamic-plugin-sdk';
+
+import { AUTOUNATTEND, UNATTEND } from '../sysprep-utils';
+
+const useSysprepConfigMaps = (namespace: string): WatchK8sResult<IoK8sApiCoreV1ConfigMap[]> => {
+  const [configmaps, configmapsLoaded, configmapsError] = useK8sWatchResource<
+    IoK8sApiCoreV1ConfigMap[]
+  >({
+    groupVersionKind: modelToGroupVersionKind(ConfigMapModel),
+    isList: true,
+    namespace,
```

Comment:
it might have problem with all namespace, did u tried?

Reply:
Good question! Well, I was testing the UI on all-namespaces project and it seemed to me working great. However I was not investigating this further due to lack of time. I'll check it now again and fix it eventually.

Reply:
So I've found the target namespace when "all-namespaces" selected is `default`, so we find all ConfigMaps with this namespace to be displayed, so we are fine.